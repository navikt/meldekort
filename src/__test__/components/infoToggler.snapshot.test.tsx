import * as React from "react";
import * as renderer from "react-test-renderer";
import ProviderWrapper from "../testSetup/providerWrapper";
import InfoToggler from "../../app/components/utvidetinformasjon/infoToggler/infoToggler";

it("InfoToggler", () => {
  const tree = renderer.create(
    <ProviderWrapper>
      <InfoToggler
        onToggle={() => {
          return 1 + 1;
        }}
      >
        <span>Test</span>
      </InfoToggler>
    </ProviderWrapper>
  );
  expect(tree).toMatchSnapshot();
});
