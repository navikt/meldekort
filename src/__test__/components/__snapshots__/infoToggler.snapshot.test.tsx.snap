// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`InfoToggler 1`] = `
<button
  aria-expanded={false}
  className="infoToggler"
  onClick={[Function]}
>
  <span
    className="infoToggler__content"
  >
    <span
      className="infoToggler__label"
    >
      <span>
        Test
      </span>
    </span>
    <span
      className="infoToggler__chevron"
    >
      <span
        className="nav-frontend-chevron chevronboks chevron--ned"
      />
    </span>
  </span>
</button>
`;
